MCMC/MCMC-flow/templates/fry.sh

```diff
@@ -0,0 +1,22 @@
+{% extends "base_script.sh" %}
+{% block header %}
+{% set gpus = operations|map(attribute='directives.ngpu')|sum %}
```

Comment:
Let's remove anything references GPUs in these templates. We can keep the wall time, partition, job_output stuff.